packages/react-heartwood-components/src/components/Button/Button.tsx

```diff
@@ -28,7 +28,7 @@ export interface IButtonProps {
 	children?: Node
 
 	/** Sets the visual appearance of the button. May be primary, secondary, simple, or caution. */
-	kind?: string
+	kind?: 'primary' | 'secondary' | 'simple' | 'caution'
```

Comment:
It'd be great to build this off of an enum that exposed these options to other components. i.e. `kind={BUTTON_KINDS.PRIMARY}`. Unrelated to the circle loader, though.